koala/ast/Range.py

```diff
@@ -0,0 +1,299 @@
+
+import re
+from collections import OrderedDict
+# from excelutils import col2num
+import string
+def col2num(col):
+    
```

Comment:
i think this method was already implemented by openpyxl and i copied it in the utils somewhere